src/main/java/bo/usfx/Example.java

```diff
@@ -1,3 +1,7 @@
+/**
+ * Este paquete contiene la clase Example,que
```

Comment:
It's not necessary to add comments. But if you want to add them you have to change everything to English.